components/script/dom/readablestreamdefaultreader.rs

```diff
@@ -211,6 +203,22 @@ impl ReadRequest {
             ReadRequest::DefaultTee { tee_read_request } => {
                 tee_read_request.close_steps(can_gc);
             },
+            ReadRequest::ReadLoop {
+                success_steps,
+                reader,
+                bytes,
+                ..
+            } => {
+                // Release the temporary reader asynchronously.
+                // avoid re-entrancy/borrow during close; queue a microtask.
```

Comment:
What was the borrow problem? 

I don't think it's correct to release the reader at a later microtask, because that would be noticeable to code in the current task(the reader is still locked).